lib/usd/hdMaya/adapters/materialNetworkConverter.cpp

```diff
@@ -31,7 +24,14 @@
 #include <maya/MPlugArray.h>
 #include <maya/MStatus.h>
 
-#include "../utils.h"
+#include <hdMaya/adapters/adapterDebugCodes.h>
+#include <hdMaya/adapters/materialAdapter.h>
+#include <hdMaya/adapters/mayaAttrs.h>
+#include <hdMaya/adapters/tokens.h>
+
```

Comment:
Remove this blank line to put all the `hdMaya` includes in a group together?

Reply:
see 7cb7810a